app/views/categories/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<section class="container mt-3">
+  <div class="row">
+    <div class="col-12">
+      <%= form_with(model: @category, url: categories_path, method: :post, local: true, class: "form") do |form| %>
+        <% if @category.errors.any? %>
+          <div style="color: red">
+            <ul>
+              <% @category.errors.each do |error| %>
+                <li><%= error.full_message %></li>
+              <% end %>
+            </ul>
+          </div>
+        <% end %>
+        <div class="field">
+          <%= form.text_field :name, placeholder: 'Name', class: 'w-100 custom-field-height form-control' %>
+          <% if form.object.errors.has_key?(:name) %>
+            <div class="text-danger"><%= form.object.errors[:name].first %></div>
+          <% end %>
+        </div>
+
+        <div class="field mt-2">
+          <%= form.label :icon, class: 'h5' %><br>
+          <%= form.file_field :icon, class: 'w-100 custom-field-height form-control' %>
+          <% if form.object.errors.has_key?(:icon) %>
+            <div class="text-danger"><%= form.object.errors[:icon].first %></div>
+          <% end %>
+        </div>
+
+        <div class="action d-flex justify-content-center mt-4">
+          <%= form.submit "Create Category", class: 'btn btn-info text-white' %>

```

Comment:
- [ ] Hi @kamranasif506, based on the requirements of this project, when the user click(or taps) the "Create Category" button, to create a new category, it should take the user to the transactions page for that category. Let's kindly fix the minor issue.

Reply:
Hello @FrederickMih 
I have read your comment on this point very carefully and cross-check it with requirement. and my implementation for navigation are same as requirements. this is what it is written in project requirement.
- When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
- There is a button "add a new category" at the bottom that brings the user to the page to create a new category.

here i attached the screenshot as well
![image](https://github.com/kamranasif506/Budget-App/assets/98281453/e0d93fa2-616c-4a1d-b361-fc84ed3038e8)